uwblueprint/ccbc
# Add getReview(id) and getReviews endpoints

## Notion ticket link

[Create Review Service](https://www.notion.so/uwblueprintexecs/Create-Review-Service-4e0fa6707a164d118aad9f914fea42a3)



## Implementation description
* Added services for getReview(id) and getReviews
* Added tests
* Added Swagger documentation

## Screenshots
GET all reviews
![2021-12-03 17 57 36](https://user-images.githubusercontent.com/44102730/144683629-42834ce6-1d60-45c7-9278-ff40a027fe1c.gif)

GET specific review
![2021-12-03 17 59 33](https://user-images.githubusercontent.com/44102730/144683688-1b299a03-0db7-49ff-acff-0416e9803722.gif)


## Steps to test
1. Run tests in ts-backend to make sure they are passing
2. Make sample GET requests to /reviews and /reviews/{id} endpoint in Postman



## What should reviewers focus on?
* 


## Checklist
